packages/base-rollup-config/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@inrupt/base-rollup-config",
+  "version": "2.1.1",
```

Comment:
Kind of annoying that the first version of this package has to be 2.1.1 cause we update all package versions symmetrically but, thats more just a small nit-pick on the versioning we use here than your code.